src/plugins/home/server/services/sample_data/data_sets/otel/logs_field_mappings.ts

```diff
@@ -0,0 +1,393 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const logsFieldMappings = {
+  body: {
+    type: 'text',
+    fields: {
+      keyword: {
+        type: 'keyword',
+        ignore_above: 256,
+      },
+    },
+  },
+  droppedAttributesCount: {
+    type: 'long',
+  },
+  flags: {
+    type: 'long',
+  },
+  instrumentationScope: {
+    properties: {
+      name: {
+        type: 'text',
+        fields: {
+          keyword: {
+            type: 'keyword',
+            ignore_above: 256,
+          },
+        },
+      },
+    },
+  },
+  log: {
+    properties: {
+      attributes: {
+        properties: {
+          address: {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+                ignore_above: 256,
+              },
+            },
+          },
+          contentRoot: {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+                ignore_above: 256,
+              },
+            },
+          },
+          context: {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+                ignore_above: 256,
+              },
+            },
+          },
+          envName: {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+                ignore_above: 256,
+              },
+            },
+          },
+          otelServiceName: {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+                ignore_above: 256,
+              },
+            },
+          },
+          otelSpanID: {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+                ignore_above: 256,
```

Comment:
many fields like IDs are `text`, is it expected? usually we have them as keyword only

Reply:
We took the mappings directly from opentelemetry repo.

Reply:
see new commit [66efe4c](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8587/commits/66efe4c6b6fc328b4dea670653a387577c7e3ba8)